elide-core/src/main/java/com/yahoo/elide/ElideSettingsBuilder.java

```diff
@@ -159,6 +171,22 @@ public ElideSettingsBuilder withUseFilterExpressions(boolean useFilterExpression
         return this;
     }
 
+    public ElideSettingsBuilder withISO8601Dates(String dateFormat, TimeZone tz) {
```

Comment:
I'm not exposing Serde's directly here as that would require more testing and the issue right now is Date formatting.  However, the plumbing is in place to expose Serde's at some future point.